syntax = "proto2";

package bchain;

import "transaction.proto";

message Block {
  optional int64 id = 1;
  optional string proof = 2;
  optional string previous_hash = 3;
  optional int64 previous_block_id = 4;
  repeated Transaction transactionList = 5;
  optional TimeStamp timestamp = 6;

  }

